### DISCUSSION TITLE
notes/00-thinkcente-minecraft-nixos

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#58] [Announcements]
# notes/00-thinkcente-minecraft-nixos

trifold learning experience with little cost

https://www.ears.sh/notes/00-thinkcente-minecraft-nixos

**Comment by 1wrnr** [comment]
test test test

**Comment by hohloh** [comment]


**Reply by 1wrnr** [reply]
ok, das ganze mit weniger glühendem provider und es passt

**Reply by 1wrnr** [reply]
| name | gewicht |
| --- | --- |
| alfons | 120kg |

**Reply by 1wrnr** [reply]
markdown funktioniert hier auch komplett, interesting
